cross/lua/patches/Makefile.patch

```diff
@@ -0,0 +1,45 @@
+--- Makefile.orig	2017-04-26 18:12:27.442079208 +0200
```

Comment:
Please use recent file naming convention, for instance "001_Makefile_add_shared_library.path"
If this patch comes elsewhere, please add a comment as patch first line with description why and original source